addOns/saverawmessage/src/main/resources/org/zaproxy/zap/extension/saverawmessage/resources/Messages.properties

```diff
@@ -1,8 +1 @@
-saveraw.file.description       = Raw
-saveraw.file.save.error        = Error saving file to {0}.
-saveraw.popup.option = Save Raw
-saveraw.popup.option.all = All
-saveraw.popup.option.body = Body
-saveraw.popup.option.header = Header
-saveraw.popup.option.request = Request
-saveraw.popup.option.response = Response
\ No newline at end of file
+saveraw.retired=The Save Raw HTTP Message add-on has been retired. This functionality is now provided by the Import/Export add-on.
```

Comment:
I'd suggest hardcoding the message, not sure worth translating, should be removed soon anyway? (We'll have to remove them from Crowdin too.)

Reply:
Yup

Reply:
They are no longer there, they were not included when Crowdin integration was restored.